staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -95,6 +96,8 @@ func (b *backend) processEvents(ev ...*auditinternal.Event) error {
 		list.Items = append(list.Items, *e)
 	}
 	return b.w.WithExponentialBackoff(func() rest.Result {
+		trace := utiltrace.New(fmt.Sprintf("Audit events webhook request for %s, event list count: %d", b.name, len(list.Items)))
+		defer trace.LogIfLong(5 * time.Second)
```

Comment:
we could scale this according to the length of list.Items

Reply:
Set the per object cost for audit at 25ms per object since the event objects can be fairly heavy. They contain a nested request and response object plus event specific fields and they're JSON serialized to the webhook.